src/plugins/data/public/search/aggs/agg_config.ts

```diff
@@ -257,7 +260,7 @@ export class AggConfig {
     return configDsl;
   }
 
-  toJSON() {
+  toJSON(): AggConfigJson {
```

Comment:
should probably not be called toJSON, but no need to rename now

Reply:
i'll create a `serialize()` method and have `toJSON()` call it internally. then we can eventually remove `toJSON`